packages/meditrak-server/src/routes/exportSurveyResponses/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+export { exportSurveyResponses } from './exportSurveyResponses';
```

Comment:
Perhaps there is no need to create a folder which contains a single exported file, but this is a detail so no need to change it... You may also have a particular folder structure in mind for all routes